### PR TITLE
ladspa-header: new formula

### PR DESCRIPTION
**Pull request by maxim-belkin**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --new-formula <formula>` (after doing `brew install <formula>`)?

-----

This formula installs `ladspa.h`. Alternative solution to having this formula would be to use a resource in formulas that need `ladspa.h`.

### CODE REVIEW

**Comment by maxim-belkin** [comment]
Check out the number of this PR! yay! :)

**Comment by maxim-belkin** [comment]
@sjackman @rwhogg:
`--keep-old` needs a fix.

File: Formula/ladspa-header.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+class LadspaHeader < Formula
+  desc "Linux Audio Developer's Simple Plugin API"
+  homepage "http://www.ladspa.org"
+  url "https://gist.githubusercontent.com/maxim-belkin/721fafe404e78e52cf3bf29a259ea9d2/raw/c50f363ec478e23e0bd5b28ea705b5ae52523abb/ladspa.h"
+  version "1.1"
+  sha256 "0c73624df742727076c65a9fec4c2806d37bcb7245f62e973d070048b6d333af"
+
+  def install
+    mkdir_p include.to_s
+    mv "ladspa.h", "#{include}/ladspa.h"
```
Comment (rwhogg) [inline Formula/ladspa-header.rb]: Does `mv "ladspa.h", include/"ladspa.h"` work?

**Comment by rwhogg** [comment]
I think I'd prefer this as a resource to be honest.

File: Formula/ladspa-header.rb (hunk 1 above)
Comment (sjackman) [inline Formula/ladspa-header.rb]: Try instead
```ruby
include.install "ladspa.h"
```

**Comment by sjackman** [comment]
I'm fine with either a LADSPA resource in `rubberband` or a formula. I'd like to know how many formula depend on this LADSPA header. If it's only `rubberband`, then a resource is the way to go.

File: Formula/ladspa-header.rb (hunk 1 above)
Comment (maxim-belkin) [inline Formula/ladspa-header.rb]: did as Shaun suggested

**Comment by maxim-belkin** [comment]
Let's do it as a resource! Closing this thread.

**Pull request closed without merging**